schemas/v3.schema.graphql

```diff
@@ -392,8 +398,12 @@ type ReserveParamsHistoryItem @entity {
   lifetimeBorrows: BigInt! # default: 0
   lifetimeLiquidated: BigInt! # default: 0
   lifetimeFlashLoans: BigInt!
-  lifetimeFlashLoanPremium: BigInt!
+  lifetimeFlashLoanPremium: BigInt! #deprecated
```

Comment:
while this is no longer needed, removing it would break integrations like cryptofees